components/cms/Faq.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div>
+    <v-expansion-panels focusable inset>
+      <v-expansion-panel
+        v-for="item in items"
+        :key="item.name"
+        :class="theme === 'dark' ? 'faq-item-dark' : 'faq-item-light' "
+      >
+        <v-expansion-panel-header
+          :class="theme === 'dark' ? 'faq-question-dark' : 'faq-question-light' "
+          class="faq-question"
+          color="primary"
+        >
+          {{ item.question }}
+        </v-expansion-panel-header>
+        <v-expansion-panel-content
+          :class="theme === 'dark' ? 'faq-answer-dark' : 'faq-answer-light' "
+          class="faq-answer"
+        >
+          {{ item.answer }}
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+  </div>
+</template>
+
+<script>
+
+import CMSMixin from '~/mixins/CMSMixin'
```

Comment:
Are you using this mixin for some reason?

Reply:
Im new in vue framework and only after add this import and variable "mixins" the theme value was found in template

Reply:
Sorry, I was thinking of another mixin from the project, you use this right :)
More information about vue mixins: https://vuejs.org/v2/guide/mixins.html